contributing/README.md

```diff
@@ -10,13 +10,38 @@ us on [Slack]. Please also take a look at our [code of conduct], which details
 how contributors are expected to conduct themselves as part of the Crossplane
 community.
 
+## Establishing a Development Environment
+
+> The Crossplane project consists of several repositories under the crossplane
+> and crossplane-contrib GitHub organisations. We're experimenting with
+> [Earthly] in this repository (crossplane) and crossplane-runtime. Most other
+> repositories use a `Makefile`. To establish a development environment for a
+> repository with a `Makefile`, try running `make && make help`.
+
+Crossplane is written in [Go]. You don't need to have Go installed to contribute
+code to Crossplane but it helps to use an editor that understands Go.
+
+To setup a Crossplane development environment:
+
+1. Fork and clone this repository.
+1. Install [Docker][get-docker] and [Earthly][get-earthly].
+
+Use the `earthly` command to build and test Crossplane. Run `earthly doc` to see
+available build targets.
+
+Useful targets include:
```

Comment:
Do you want to specifically mention `earthly +build` here? I had to look through the `Earthfile` to find that one. It is probably a convention to use `+build` as the target, so maybe folks already know that, but it's probably nice to be explicit about it since so many of us will be new to this tooling.